### PR TITLE
[BFD-751] Add Unit Tests to DME EOB Profile

### PR DESCRIPTION
**Pull request by JeremyIR**
### Change Details

Modify DME v2 transform test code as an expansion similar to Outpatient claim transformer tests, so that there is more confidence individual fields and values are working as intended.

### Acceptance Validation

- Confirm that the same level of and unit tests for BFD V2 are applied to V2
- Ensure that all required CARIN data fields have corresponding unit tests for V2

### Feedback Requested

Do the updated set of tests for DME v2 Transformer thoroughly account for the CARIN fields?

### External References

- [BFD-751](https://jira.cms.gov/browse/BFD-751)

### Security Implications

- [ ] new software dependencies

- [ ] altered security controls

- [ ] new data stored or transmitted

- [ ] security checklist is completed for this change

- [ ] requires more information or team discussion to evaluate security implications

### CODE REVIEW

File: apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/DMEClaimTransformerV2Test.java [hunk 1]
```diff
@@ -1,23 +1,50 @@
...
+import org.hl7.fhir.r4.model.ExplanationOfBenefit.Use;
+import org.hl7.fhir.r4.model.Extension;
+import org.hl7.fhir.r4.model.Identifier;
+import org.hl7.fhir.r4.model.Money;
+import org.hl7.fhir.r4.model.Period;
+import org.hl7.fhir.r4.model.Quantity;
+import org.hl7.fhir.r4.model.Reference;
+import org.junit.Assert;
+import org.junit.Before;
 import org.junit.Ignore;
```
Comment (mackec1) [inline apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/DMEClaimTransformerV2Test.java:41]: Jeremy; just a heads up for future ref. We've been trying to have a test where we output the JSON to the termin. We mark that test with @Ignore but if a tester ever wants to look at the JSON payload they simply comment out the @Ignore (i.e., //@Ignore).

Comment (rtravitz) [reply apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/DMEClaimTransformerV2Test.java:41]: Those do make it easier for me to get a payload when reviewing.

Comment (JeremyIR) [reply apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/DMEClaimTransformerV2Test.java:41]: So there's the Serialize SampleA test in this file. I used that to aid writing the tests. @mackec1 can you elaborate a bit on what test you are referring to?

Comment (mackec1) [reply apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/DMEClaimTransformerV2Test.java:41]: Nope, that's fine the way you have it.

**Review by mackec1 — APPROVED** [review]
LGTM.

**Review by rtravitz — APPROVED** [review]
Nice work 🚀

**Pull request merged**